graspologic/utils/utils.py

```diff
@@ -55,14 +57,18 @@ def import_graph(graph, copy=True):
         out = check_array(
             graph,
             dtype=[np.float64, np.float32],
+            accept_sparse=isspmatrix(graph),
```

Comment:
would `True` achieve the same effect?

Reply:
I'm not sure how this would act if `check_array` is given an nd.array and `accept_sparse` is `True`; I can check though.

Reply:
i think it still works and would be slightly more clear, let me know if not